src/AppInstallerCommonCore/Public/AppInstallerRuntime.h

```diff
@@ -112,10 +112,10 @@ namespace AppInstaller::Runtime
 
     // Gets the PathDetails used for the given path.
     // This is exposed primarily to allow for testing, GetPathTo should be preferred.
-    PathDetails GetPathDetailsFor(PathName path);
+    PathDetails GetPathDetailsFor(PathName path, bool anonymize = false);
 
     // Gets the path to the requested location.
-    std::filesystem::path GetPathTo(PathName path);
+    std::filesystem::path GetPathTo(PathName path, bool anonymize = false);

```

Comment:
nit: I don't think name anonymize fits here very well. Anonymize is quite context specific. Maybe bool forDisplay?